analysed-packages/tslib/version-1.22/tslib-1.22-OSS-disclosure.txt

```diff
@@ -0,0 +1,3071 @@
+Fifth Floor, Boston, MA 02110-1301, USA.
+
+Also add information on how to contact you by electronic and paper mail.
+
+If the program is interactive, make it output a short notice like this when it
+starts in an interactive mode:
+
+Gnomovision version 69, Copyright (C) year name of author Gnomovision comes with
+ABSOLUTELY NO WARRANTY; for details type `show w'. This is free software, and
+you are welcome to redistribute it under certain conditions; type `show c' for
+details.
+
+The hypothetical commands `show w' and `show c' should show the appropriate
+parts of the General Public License. Of course, the commands you use may be
+called something other than `show w' and `show c'; they could even be mouse-
+clicks or menu items--whatever suits your program.
+
+You should also get your employer (if you work as a programmer) or your school,
+if any, to sign a "copyright disclaimer" for the program, if necessary. Here is
+a sample; alter the names:
+
+Yoyodyne, Inc., hereby disclaims all copyright interest in the program
+`Gnomovision' (which makes passes at compilers) written by James Hacker.
+
+<signature of Ty Coon>, 1 April 1989 Ty Coon, President of Vice
+
+This General Public License does not permit incorporating your program into
+proprietary programs. If your program is a subroutine library, you may consider
+it more useful to permit linking proprietary applications with the library. If
+this is what you want to do, use the GNU Lesser General Public License instead
+of this License.
+
+License text of the Libtool-exception as an exception to the GPL-2.0-or-later:
+
+As a special exception to the GNU General Public License, if you distribute this
+file as part of a program or library that is built using GNU Libtool, you may
+include this file under the same distribution terms that you use for the rest of
+that program.
+
+--------------------------------------------------------------------------------
+
+License-of-GNU-Licenses
+
+Everyone is permitted to copy and distribute verbatim copies
+of this license document, but changing it is not allowed.
+
+--------------------------------------------------------------------------------
+
+FSFULLRWD
+
+This Makefile.in is free software; the Free Software Foundation gives unlimited
+permission to copy and/or distribute it, with or without modifications, as long
+as this notice is preserved.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY, to the extent permitted by law; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+--------------------------------------------------------------------------------
+
+FSFULLR
+
+This file is free software; the Free Software Foundation gives unlimited
+permission to copy and/or distribute it, with or without modifications, as long
+as this notice is preserved.
+
+--------------------------------------------------------------------------------
+
+FSFUL
```

Comment:
Please add here additionally the following text for the file configue on root level:
This config.status script is free software; the Free Software Foundation
gives unlimited permission to copy, distribute and modify it.
Because this phrase is also present in the file (quite at the end of the file)